src/main/resources/application.yml

```diff
@@ -80,6 +80,7 @@ logging:
 app:
   brokerEncryptPassword: "\${BROKER_ENCRYPT_PASSWORD:ENCR_PSW}"
   organizationEncryptPassword: "\${ORGANIZATION_ENCRYPT_PASSWORD:ENCR_PSW}"
+  orgSilServiceEncryptPassword: "\${ORG_SIL_SERVICE_ENCRYPT_PASSWORD:ENCR_PSW}"
```

Comment:
if we choose to use a different key, we should:
* make a PR on p4pa-infra to generate and store on the keyvault the new key
* make a PR on aks-deploy repo to define the value for the new environment variable
* update the README with the ENV variable
* update Confluence page (https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1190330377/PU+-+Secret+manuali) to describe the new manual secret and the way to obtain a new value

otherwise, we could also use the same encryption key used for the organization entity